go-controller/pkg/cni/ovs.go

```diff
@@ -172,14 +172,28 @@ func getIfaceOFPort(ifaceName string) (int, error) {
 	return iPort, nil
 }
 
-type openflowQuery struct {
-	match  string
-	tables []int
-}
+func doPodFlowsExist(mac string, ifAddrs []*net.IPNet, ofPort int) bool {
```

Comment:
Wait why do we still need to manually make sure the ovs physical rules are there if `ovn-installed` works again?